src/styles/main.scss

```diff
@@ -0,0 +1,4 @@
+@import "variables";
```

Comment:
Consider checking if the imported variables file contains CSS variables for stopwatch, such as colors, sizes, and durations.